mappings/net/minecraft/entity/effect/StatusEffect.mapping

```diff
@@ -10,6 +10,8 @@ CLASS net/minecraft/class_1291 net/minecraft/entity/effect/StatusEffect
 		ARG 0 type
 	METHOD method_5555 onApplied (Lnet/minecraft/class_1309;Lnet/minecraft/class_1325;I)V
 	METHOD method_5556 getColor ()I
+	METHOD method_5559 getTranslationKey ()Ljava/lang/String;
```

Comment:
wait my bad... this is not a translation key

Reply:
```suggestion
	METHOD method_5559 loadTranslationKey ()Ljava/lang/String;
```
Dunno why but mojang just decided to create this method and have another public method that actually offers the translation key which calls this method..

Reply:
Typically don't we use getXThing for that type of method?

Reply:
There is already a `getTranslationKey` method with same signature (method_5567)
and that 5567 just calls this method